swagger-ui/swagger.yaml

```diff
@@ -441,6 +443,12 @@
             required: true
             type: string
             x-example: xrp
```

Comment:
Collateral denom has been replaced by collateral type -so we don't need both here.

Reply:
Gotcha

Reply:
https://github.com/Kava-Labs/kava/pull/653/commits/402223a6ab1c63ed0789c72e10d4d40ef2811a4f